### DISCUSSION TITLE
FR: add ability to move application assigned to disconnected device

### DISCUSSION THREAD

**Discussion by alefranz** [File-New-Project/EarTrumpet#630] [Ideas]
Currently if you assign an application to a device and then disconnect or disabled that device. that application would fail to reproduce any audio and it's up to the application to display a meaningful error (e.g. Spotify will just skip endlessly through all the songs in catalog 😅).

It would be nice to have the ability in the mixed to see applications assigned to missing devices so that you can move it back to one available.

Ideally, it would be nice to this to happen automatically, and the application would then move again to the assigned device when it is reconnected.

Thank you!

**Comment by ryantheleach** [comment]
As an aside, I would NOT want them to automatically move, unless they ALSO automatically moved back.

Having audio blasting through speakers accidentally when something is disconnected is bad enough, HOWEVER I can see this being a UX issue for people who expect their applications to just work, so could concede that switching is better behavior.

But if it forgot it's original settings, and didn't switch back, that would be really irritating when rearranging cables and stuff on desks.
